pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-searching-modules.adoc

```diff
@@ -8,7 +8,7 @@ ifdef::context[:parent-context: {context}]
 :context: search-modules
 
 [role="_abstract"]
-The *{ContentTerm}* tab allows you to search for content in {ProductShortName} by simple keyword or free text search, and by applying a number of different filters to search and to sort the results.
+You can search for content in the *{ContentTerm}* tab of {ProductShortName} by simple keyword or free text search after you  apply one or more filters to your query.
```

Comment:
"one or more filters" I only see one filter, and I have no control over it. @noopurmath3 Will that change anytime soon?

Reply:
There's the Repo filter on the left and the Status and Content type filters at the top. Should i list them to make it clearer?

Reply:
@stoobie As @RichardHoch mentioned, we have Repo filter as well as the Status and Content Type filters as of now.

Reply:
@noopurmath3 Yes, that's true,  but there's only one filter that is actually labeled as a filter in the UI. @RichardHoch I think it could be helpful to mention the different filters here. Just "You can filter by repo, content type, etc."